Automattic/sensei
# Remove screenshot uploads of e2e failed tests

Related PR: https://github.com/Automattic/sensei/pull/3280

### Changes proposed in this Pull Request

* Remove screenshot uploads. We're keeping the screenshots in the same folder for test when running locally and in the Travis it'll send through Slack only.

### Testing instructions

* Check this example Job: https://travis-ci.org/github/Automattic/sensei/jobs/699344466 to check if the screenshots weren't uploaded.
* Access the Slack and make sure the errors were sent there.
